src/main/java/com/google/firebase/messaging/FirebaseMessaging.java

```diff
@@ -90,7 +90,9 @@ public String send(@NonNull Message message) throws FirebaseMessagingException {
    * Sends the given {@link Message} via Firebase Cloud Messaging.
    *
    * <p>If the {@code dryRun} option is set to true, the message will not be actually sent. Instead
-   * FCM performs all the necessary validations, and emulates the send operation.
+   * FCM performs all the necessary validations, and emulates the send operation. {@code dryRun} 
+   * option is useful for validating if an FCM registration has been deleted. But it cannot be used
```

Comment:
"validating if" seems odd to me . . . how about "determining whether". ?

Reply:
Done.